src/java/org/apache/cassandra/schema/DroppedColumn.java

```diff
@@ -57,6 +57,11 @@ public boolean equals(Object o)
         return column.equals(dc.column) && droppedTime == dc.droppedTime;
     }
 
+    public boolean equalsWithoutKsTb(DroppedColumn other)
```

Comment:
@Maxwell-Guo is this still necessary as we are not dealing with dropped columns in this PR?